index.html

```diff
@@ -237,5 +234,77 @@ <h1>Skills</h1>
       </div>
     </section>
     </div>
+    <!----last section-->
+ <section class="details">
+
+    <div class="contact-p">
+         <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+    </div>
+    <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- I get an error when I try to submit form details on your site. This is because you have not used a valid form_id. I'd suggest you signup on and use your given id [formspree](https://formspree.io/html). Please kindly fix this 👍 👍